package.json

```diff
@@ -83,7 +83,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "lint-staged": "lint-staged"
```

Comment:
I think it's not necessary. It's just a task for launching lint-staged itself. But what is the reason to use it?

Reply:
Trying to resolve executaion permissions issue

Reply:
<img width="674" alt="Screenshot 2021-07-01 at 13 02 30 (1)" src="https://user-images.githubusercontent.com/2698505/124118239-c838dd80-da79-11eb-8a68-d0f2b0907de6.png">